packages/vertexai/src/methods/chrome-adapter.test.ts

```diff
@@ -107,51 +100,6 @@ describe('ChromeAdapter', () => {
         })
       ).to.be.false;
     });
-    it('returns false if request system instruction has function role', async () => {
```

Comment:
We used to copy the systemInstruction out of the request, but removed that in favor of onDeviceParams.systemInstruction, which isn't currently validated. Added backlog item to revisit this.